src/sleep/sleep.c

```diff
@@ -264,10 +264,16 @@ static int execute(
 
 static int execute_s2h(const SleepConfig *sleep_config) {
         _cleanup_hashmap_free_ Hashmap *last_capacity = NULL, *current_capacity = NULL;
+        _cleanup_(freeze_thaw_user_slice) const char *method = "thaw";
         int wakeup_type_bit = 0, r;
 
         assert(sleep_config);
 
+        method = "freeze";
```

Comment:
This means the cleanup will freeze again

Reply:
ok. so initially to freeze, i can then directly pass `freeze` to function `freeze_thaw_user_slice`